expr/func_removeemptyseries.go

```diff
@@ -42,9 +41,6 @@ func (s *FuncRemoveEmptySeries) Exec(dataMap DataMap) ([]models.Series, error) {
 
 	var output []models.Series
 	for _, serie := range series {
-		serie.Target = fmt.Sprintf("removeEmptySeries(%s, %g)", serie.Target, s.xFilesFactor)
```

Comment:
With the default being set in the constructor, we might not need the default override in line 39